neuror/unravel.py

```diff
@@ -93,6 +93,12 @@ def _unravel_section_path_length(sec, window_half_length, soma, legacy_behavior)
         window_start = np.argmin(abs(_l_left - window_half_length))
         window_end = np.argmin(abs(_l_right - window_half_length))
 
+        if window_start != window_end:
+            raise Exception(
+                f"""we cannot unravel with path length {window_half_length},
+                consider resampling as there are points to far from each others."""
```

Comment:
```suggestion
                consider resampling as there are points too far from each others."""
```

Reply:
We should probably also recommend the tool to do this resampling, so people know that it exists.

Reply:
I am a little sceptical of this approach. Shouldn't the algorithm work correctly regardless of the point sampling?

How does the user know which is the "ideal" density so that the algorithm works properly?

wdyt?

Reply:
When I said correctly, I didn't mean to not run at all. I was wondering if it would be a bad idea to introduce resampling if necessary so that unraveling is always correct.

And I want to underline that this is a question, not a suggestion. I don't how unreaveling is in general used and if this would be a good thing to do.